.travis.yml

```diff
@@ -13,7 +13,7 @@ notifications:
   webhooks:
     - http://savage.nonblocking.io:8080/savage/travis
 before_install:
-  - pip install --user gsutil protobuf urllib3[secure]
+  - pip install --user protobuf urllib3[secure]
```

Comment:
IIRC, `urllib3[secure] was only needed because it was a dependency of `gsutil`. Can you try removing it?

Future task: Can we also use `apt` to install `protobuf` too? If so, we can get rid of `pip`, which is somewhat troublesome to maintain and cache.

Reply:
Looks like that worked!